pdfjam-help.txt

```diff
@@ -101,6 +101,11 @@ where
                   Provide text for the  Title, Author, Subject and Keywords
                   in the output PDF file.  Requires the  LaTeX 'hyperref'
                   package.  These options, individually, over-ride --keepinfo.
+     --longedge
+     --shortedge
+		  Specify the longedge rotation of paper. This is probably
```

Comment:
I think we can improve this explanation. How about:

```
Specify the direction in which double-sided pages are flipped. It may not work correctly in all cases, so if every other page is rotated 180°, flip this switch.
```